### ISSUE TITLE
Deckbrew no longer offers pricing.

### ISSUE THREAD

**Issue by Psithief** [Cockatrice/Cockatrice#548]
[This is unfortunate.](http://deckbrew.com/api/)

> Due to a misundestanding between TCGPlayer and Deckbrew, prices are no longer available in the API. To prevent breaking existing applications, all prices have been set to zero.

**Comment by woogerboy21** [comment]
Boo.... I wonder if http://www.mtggoldfish.com/ has any type of api type interface?

**Comment by ZeldaZach** [comment]
Wow that's not cool. What can we do now?

**Comment by Daenyth** [comment]
Without an api source, not much. Maybe we could approach TCGPlayer and ask for access to the info?

**Comment by ZeldaZach** [comment]
We have to provide a reason for access... which I don't think they'd be willing to give us it :P

**Comment by Psithief** [comment]
> approach TCGPlayer

It's not in their interests to agree.

**Comment by ctrlaltca** [comment]
hum.. i suppose they'd want a "buy this card on TCG" button on every card..

**Comment by tooomm** [comment]
Maybe a "Buy deck on TCG" next to "Analyze deck on deckstats.net" will do it?

**Comment by skwerlman** [comment]
deckstats.net provides card value info; would it be hard to parse their results page for the pricing info?

**Comment by ZeldaZach** [comment]
That's a possibility... maybe they have an API service

**Comment by ctrlaltca** [comment]
Since trying to fetch card prices can now result in errors, as an alternative i propose to temporary mae the "fetch card prices" button execute the "analyze deck" action.

**Comment by Daenyth** [comment]
We should just disable the menu item so it can't be activated
